src/main/rules/GCI95/GCI95.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
in this PR, you should put the modification of RULES.md to add inside this rule in the array